---
# Title, summary, and page position.
linktitle: About
summary: 
weight: 1
icon: book
icon_pack: fas

# Page metadata.
title: About
date: ''
type: book # Do not modify.
---

[BnaOmics](https://bnaomics.ocri-genomics.net/) is a comprehensive and
multi-omics database for Brassica napus genetics discovery and crop
improvement. It integrates omics data generated from 381 germplasms
(from our lab) and 2504 germplasms (downloaded from public databases).
BnaOmics includes:

-   different versions of genome assemblies and annotations of 18
    germplasms (total of 26 genomes), including Darmor, Express617,
    Gangan, HTR001, Kale, Laurentian, Mendel, NY7, No2127, Quinta,
    Shengli, Tapidor, Westar, ZS11, ZS4-2, ZS9, ZY821, Zheyou7. For the
    details of source reference of each genome, users can refer to
    [here](https://bnaomics.ocri-genomics.net//tools/jb).
-   genetic variations including single-nucleotide polymorphisms (SNPs),
    small insertions and deletions (InDels) and structure variations
    (SVs) generated from 381 and 2885 re-sequencing germplasms.
-   population gene expression of 219 accessions in shoot apical
    meristem (SAM) and silique tissues; gene expression in 1234
    libraries and 9 different tissues under pan-genome context covering
    3 distinct developmental stages during its life cycle, including
    callus, cotyledon, root, stem peel, leaf, bud, flower, silique,
    silique wall and seed.
-   trait values from 381 accessions.
-   the epigenome information, including DNA methylation, chromatin
    interaction and histone modification.

BnaOmics generated comprehensive functional annotations for each gene
model in each genome assembly, and also provide useful tools for users
to search, analyze and visualize all these data.
